src/utils/modal.ts

```diff
@@ -0,0 +1,33 @@
+import Swal from 'sweetalert2';
+import { customColor } from 'src/constants/index';
+
+interface Fn {
+  (): void;
+}
+
+export const noBtn = (text: string, fn?: Fn): void => {
+  Swal.fire({
+    position: 'top-end',
+    icon: 'success',
+    title: text,
+    showConfirmButton: false,
+    confirmButtonColor: customColor.brownDark,
+    timer: 1500,
+  });
+  fn && fn();
```

Comment:
이런 경우에는 옵셔널 체이닝으로 간단하게 작성할 수 있습니다~
```suggestion
  fn?.();
```

Reply:
옵셔널 체이닝으로 
let elem = null;
let value = elem?.value;

이렇게 변수에만 사용할수 있는 줄 알았는데 함수에도 사용할수 있었구나 싶네요
감사합니다 ㅎㅎ